platform/darwin/src/MGLFillStyleLayer.h

```diff
@@ -11,20 +12,41 @@ typedef NS_ENUM(NSUInteger, MGLFillStyleLayerFillTranslateAnchor) {
 
 @interface MGLFillStyleLayer : MGLStyleLayer
 
-// Paint properties
+#pragma mark - Accessing the Paint Attributes
 
+/**
+  Whether or not the fill should be antialiased.
+*/
 @property (nonatomic) BOOL fillAntialias;
 
+/**
+  The opacity of the entire fill layer. In contrast to the fill-color, this value will also affect the 1px stroke around the fill, if the stroke is used.
```

Comment:
We need to avoid saying “pixel” when we mean “point”, since a point is 2–3 pixels on a Retina display. This is one area where Apple platforms differ from the other Mapbox GL platforms.